internal/httphandler/users_request_handler_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"errors"
+	"io/ioutil"
```

Comment:
from the fact the encoding/json is still here I found that we still use the json package to marshal the response errors. I think it would be prudent to get rid of all such usage and use protojson exclusively

Reply:
In the protojson, we can only pass proto messages. The http error is not a proto message. We discussed this in one of the scrum call and decided to use local structure for errors in each language.